src/modules/core/components/Fields/ForceToggle/ForceToggle.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { PopperOptions } from 'react-popper-tooltip';
+import { MessageDescriptor } from 'react-intl';
+
+import { Toggle } from '~core/Fields';
+
+import { SimpleMessageValues } from '~types/index';
+
+import styles from './ForceToggle.css';
+
+const displayName = 'ForceToggle';
+
+interface Appearance {
+  theme?: 'primary' | 'danger';
+}
+
+interface Props {
+  appearance?: Appearance;
```

Comment:
You are not passing appearance from anywhere, so I would just remove it. I'm on the side of making it minimal and then adding stuff as needed. Same with all other props really except tooltipPopperOptions and obviously disabled....

But your